docs/CHANGELOG.md

```diff
@@ -14,7 +27,8 @@ The CHANGELOG for the current development version is available at
 
 ##### New Features
 
-- Added support for [AlphaFolds 200M+ structures](https://www.deepmind.com/blog/alphafold-reveals-the-structure-of-the-protein-universe) via `PandasMmcif().fetch_mmcif(uniprot_id='Q5VSL9', source='alphafold2-v3')` and `PandasPdb().fetch_pdb(uniprot_id='Q5VSL9', source='alphafold2-v3')`.
+- Added support for [AlphaFolds 200M+ structures](https://www.deepmind.com/blog/alphafold-reveals-the-structure-of-the-protein-universe) via `PandasMmcif().fetch_mmcif(uniprot_id='Q5VSL9', source='alphafold2-v3')` and `PandasPdb().fetch_pdb(uniprot_id='Q5VSL9', source='alphafold2-v3')`. (Via [Arian Jamasb](https://github.com/a-r-j), PR #[102](https://github.com/rasbt/biopandas/pull/102/files))
```

Comment:
Awesome PR! Btw in the changelog here, it should mention the new ESMFold support alongside alphafold, correct?

Reply:
I suppose we could, but I think the AF2 support has already been released so kept them separate

Reply:
Ah sorry, I didn't mean to change the comment but to add the ESMfold note below.